test/specs/pushAndPull.e2e.ts

```diff
@@ -24,7 +24,7 @@ describe('Push and Pull', async () => {
 
   step('Set up the testing environment', async () => {
     testSetup = new TestSetup('PushAndPull', false);
-    await testSetup.setUp('Enterprise');
+    await testSetup.setUp();
```

Comment:
I'm not sure why PushAndPull used to create Enterprise scratch orgs.  All the other E2E tests create Developer scratch orgs (which is the default), so I changed PushAndPull to also do the same.  The test continues to run successfully after making this switch.